kazoo_documents/src/kzd_module_utils.erl

```diff
@@ -30,6 +31,26 @@
        ,kapps_config:get_binary(?CROSSBAR_CONFIG_CAT, <<"stability_level">>)
        ).
 
+%%------------------------------------------------------------------------------
+%% @doc Get a mapping of all setter functions (start with set_) with arity 2 for
+%% a document.
+%% Returns map where the key (Function name without set_) is a binary and the
+%% value (Function name) is an atom.
+%% @end
+%%------------------------------------------------------------------------------
+-spec get_setters(atom()) -> map().
```

Comment:
Can you spec the `map()`? Maybe `#{kz_term:ne_binary() => atom()}` or whatever it should be?

Reply:
👍